app/service/model/language.py

```diff
@@ -7,6 +7,18 @@
 
 class LanguageLib:
 
+    def __init__(self):
+        self.languages = [{'name': 'English', 'code': 'en', 'html_code': 'translator-lang-option-en-US'},
```

Comment:
this should be instead inside the bootstrap. same thing with that of the workspace